Formula/nuspell.rb

```diff
@@ -6,19 +6,28 @@ class Nuspell < Formula
 
   depends_on "catch2" => :build
   depends_on "cmake" => :build
+  depends_on "p7zip" => :build
   depends_on "boost"
+  depends_on :macos => [:catalina, :high_sierra, :dunno]
```

Comment:
if it doesn't build on all supported macOS versions it should be in a tap and not in homebrew-core.

Reply:
Ah, you wrote me not to build for mojave, at least, that is how I read that. The problem for mojave is in the ruby dependency, not nuspell. I will remove that line so the error pops up in the CI.

Reply:
As you can see, 10.14 fails on https://github.com/Homebrew/homebrew-core/pull/50685/checks?check_run_id=470015012#step:4:432 Do you know how that can be fixed?

Reply:
Can you also see why continuous-integration/jenkins/ghprb has failed? Looks like an issue with GitHub.

Reply:
For 10.14, you might need either `ENV.delete("SDKROOT")` or [something like this](https://github.com/Homebrew/homebrew-core/blob/0745f27ed2eee8a8f62248938a8b8acdbd6002c2/Formula/weechat.rb#L37-L39). It's something I need to look into more at some point.

Reply:
@Bo98 Thanks for the advice. Is that only for mojave `==` or for mojave and above `>=`?

Reply:
Either. It currently makes no difference on Catalina - it will do when Xcode 12 is released. But I hope to have fixed it properly by then so these workarounds aren't needed.

Reply:
Thanks, do you have an issue on that which I can follow?